src/client/app/components/about.ts

```diff
@@ -29,9 +37,28 @@ import { environment, storageSize, Strings, storage } from '../helpers';
     `
 })
 
-export class About {
+export class About implements AfterViewInit {
     @Input() show: boolean;
     @Output() showChange = new EventEmitter<boolean>();
+
+    constructor(
+        private _effects: UIEffects,
+        private _changeDetector: ChangeDetectorRef
+    ) { }
+
+    ngAfterViewInit() {
+        if (environment.current.config.name === config['local'].name) {
+            this.configs.push({ name: 'local', value: 'Localhost' });
+        }
+        this.configs.push(
+            { name: 'production', value: 'Production' },
+            { name: 'edge', value: 'Beta' },
```

Comment:
Um... that's the reverse.

Production = production
insiders = beta (still fairly baked)
edge = gets latest from master, is the least stable.  hence alpha